### PR TITLE
update pid_tuning page for 4.3

### PR DESCRIPTION
**Pull request by ctzsnooze**
Early draft of some tidying up ideas for the PID tuning page.

- Tooltips for PID parameters simplified, hopefully easier to read.
- New tooltips for recent parameters
- Help icons for PID titles removed, the name is now the pop-up
- Motor output limit and CellCount was on the left with cell count, in a block of it's own, now moved to right.
- Advanced PID options rearranged a bit more logically, eg throttle and motor settings grouped together, with dividers
- Moved the Feedforward and Dmax labels to the white space on left

To do...
- backwards compatibility check, especially D
- antigravity 'always on' toggle into left column
- hide certain 'advanced' elements unless in expert mode
- on/off toggles for motor output limit, throttle boost etc, just to be consistent

This is how it looks:

Screen Shot 2021-11-30 at 14 04 41

### CODE REVIEW

File: src/tabs/pid_tuning.html
Comment (Asizon) [inline src/tabs/pid_tuning.html:822]: Cell count is an adition for motor output limit, so they should go always together.

**Review by asizon — CHANGES_REQUESTED** [review]

**Comment by Asizon** [comment]
Also please @ctzsnooze , consider stop making more PRs for 4.3. All of this changes are not bugfixes and can go in a future 18.1 or 19.0 versions.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
My understanding is that:
- cellCount relates to a method to automatically choose a Profile based on cell count
- motor_output_limit is a per-Profile method of attenuating motor output.
- the two are, as far as I'm aware, completely independent.

I hope someone else can clarify this.  If they are indeed linked, then of course I'll ensure the appropriate links are made.

And fear not, we are in a feature freeze from now on, and I intend to fully honour that.

File: src/tabs/pid_tuning.html
Comment (ctzsnooze) [reply src/tabs/pid_tuning.html:822]: I think they are independent.  We will get confirmation of this hopefully from other experts.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/tabs/pid_tuning.html
Comment (ctzsnooze) [reply src/tabs/pid_tuning.html:825]: We have confirmed that they are independent.

File: locales/en/messages.json
Comment (klutvott123) [inline locales/en/messages.json:1904]: I think we want just <b>F</b>eedforward

File: src/tabs/pid_tuning.html
Comment (klutvott123) [inline src/tabs/pid_tuning.html:1531]: 😮

**Review by klutvott123 — DISMISSED** [review]

**Comment by haslinghuis** [comment]
Did you check on Android how the second column displays?

File: locales/en/messages.json
Comment (haslinghuis) [reply locales/en/messages.json:1904]: <html> tags are converted here

File: src/tabs/pid_tuning.html
Comment (haslinghuis) [reply src/tabs/pid_tuning.html:1531]: Oops

File: locales/en/messages.json
Comment (klutvott123) [reply locales/en/messages.json:1904]: I mean we want "Feedforward", not "FeedForward"

Comment (ctzsnooze) [reply locales/en/messages.json:1904]: Thanks will change throughout - three instances.

File: src/tabs/pid_tuning.html
Comment (ctzsnooze) [reply src/tabs/pid_tuning.html:1531]: Oops indeed - sorry, and thank you!

**Comment by ctzsnooze** [comment]
> Did you check on Android how the second column displays?

No.  I am 100% not a CSS expert.  I just tried to demonstrate how I'd like the pid_tuning tab to display on desktop computers.  

The PR is mostly an attempt to visualise the kind of improvements in the pid_tuning tab layout, appearance and tooltips so that it works great and looks great in 4.3.  

What I'm doing here is more a conceptual draft stage. It isn't coded with strong consideration for how it might work on other platforms.  if anyone had the time to suggest how to better do the CSS that would be fantastic, I am not expert in CSS at all.

Nor have I tested the tooltip changes against 4.2.  I have focused on making sure they are appropriate for 4.3.  Some changes will be needed for 4.2 compatibility, probably, since some 4.3 tooltip changes may not make sense in 4.2, requiring a different one for 4.3.

I am sure there are better ways to do quite a lot of things, but I just needed to 'see' how things 'could' work in a 4.3 environment.  To me this a kind of conceptual model at this point.

That's why it's tagged for discussion.  If you agree, after downloading and trying it, that the old layout looks clunky, then I would really like to go ahead with these changes.

File: locales/en/messages.json
Comment (KarateBrot) [inline locales/en/messages.json:1873]: > [...] In 4.3, transition is not recommended, as it **hasoto** been replaced by the jitter reduction function. [...]

Little typo: "hasoto"

**Comment by haslinghuis** [comment]
On 4.2.9
Screenshot from 2021-11-30 01-50-19

**Comment by haslinghuis** [comment]
Needs rebase on master (conflict)

File: src/css/tabs/pid_tuning.css
Comment (Asizon) [inline src/css/tabs/pid_tuning.css:913]: Wrong indendation

**Review by asizon — DISMISSED** [review]

**Comment by Asizon** [comment]
Definitelly, this PR breaks android styles aligments. 
Screenshot_20211130_111010_com betaflight betaflight_configurator_debug

**Comment by ctzsnooze** [comment]
@Asizon @haslinghuis @limonspb  could you guys please look at the CSS, and let me know how best to provide about a ⅔ : ⅓ column split on computer screens, yet not break android views?

**Comment by ctzsnooze** [comment]
With help from @limonspb this should now work properly on Android.
Column widths tweaked.
Dmax and Derivative column labels and messages consistent across versions.

File: src/css/tabs/pid_tuning.css
Comment (ctzsnooze) [reply src/css/tabs/pid_tuning.css]: indentation fixed, and fixes for Android done

**Review by limonspb — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Comment by limonspb** [comment]
i removed "TESTED", need final testing especially on Android plz @Asizon

**Review by asizon — APPROVED** [review]

**Pull request merged**
